modules/obam/manifests/init.pp

```diff
@@ -18,11 +18,11 @@
 
   include ob_common
 
-  # Copy relevant deployment.toml to installed directory according to the spec
+  # Copy deployment.toml to installed directory
   file { "${carbon_home}/${toml_file_path}/${toml_file_name}":
     ensure  => file,
     mode    => '0644',
-    content => template("${module_name}/carbon-home/${toml_file_path}/${spec}/${toml_file_name}.erb"),
+    content => template("${module_name}/carbon-home/${toml_file_path}/${toml_file_name}.erb"),
```

Comment:
In OB 200, we used ${toml_file_path} and ${toml_file_name} as two separate variables to cater the spec based selection. Since, we don't need to cater this for accelerator, we can use only one variable to give the toml file path. Just a suggestion.

Reply:
Done.